src/main/java/edu/kis/powp/command/OperateToCommand.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand {
+    private int x;
+    private int y;
+    private Job2dDriver driver;
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).